android/java/org/chromium/chrome/browser/crypto_wallet/fragments/TransactionDetailsSheetFragment.java

```diff
@@ -0,0 +1,223 @@
+                mExecutor, mHandler, icon, mWalletListItemModel.getAccountInfo().address, true);
+        updateTxHeaderDetails(view);
+        updateTxDetails(view);
+    }
+
+    private void updateTxHeaderDetails(View view) {
+        if (JavaUtils.anyNull(mWalletModel)) return;
+        TextView fromTo = view.findViewById(R.id.from_to);
+        ImageView icon = view.findViewById(R.id.account_picture);
+        TextView txType = view.findViewById(R.id.tx_type);
+        TextView amountFiat = view.findViewById(R.id.amount_fiat);
+        TextView amountAsset = view.findViewById(R.id.amount_asset);
+
+        Utils.setBlockiesBitmapResource(
+                mExecutor, mHandler, icon, mWalletListItemModel.getAccountInfo().address, true);
+
+        if (mTxInfo.originInfo != null && URLUtil.isValidUrl(mTxInfo.originInfo.originSpec)) {
+            TextView domain = view.findViewById(R.id.domain);
+            domain.setVisibility(View.VISIBLE);
+            domain.setText(Utils.geteTLD(
+                    new GURL(mTxInfo.originInfo.originSpec), mTxInfo.originInfo.eTldPlusOne));
+        }
+
+        if (mParsedTx.getType() == TransactionType.ERC20_APPROVE) {
+            txType.setText(String.format(
+                    getResources().getString(R.string.activate_erc20), mParsedTx.getSymbol()));
+        } else if (mParsedTx.getIsSwap()) {
+            txType.setText(getResources().getString(R.string.swap));
+        } else if (mParsedTx.isSolanaDappTransaction) {
+            txType.setText(R.string.brave_wallet_approve_transaction);
+        } else {
+            txType.setText(getResources().getString(R.string.send));
+        }
+
+        if (mParsedTx.isSolanaDappTransaction) {
+            AndroidUtils.gone(amountFiat, amountAsset);
+        } else {
+            amountFiat.setText(
+                    String.format(getResources().getString(R.string.crypto_wallet_amount_fiat),
+                            String.format(Locale.getDefault(), "%.2f", mParsedTx.getFiatTotal())));
+            String amountText =
+                    String.format(getResources().getString(R.string.crypto_wallet_amount_asset),
+                            mParsedTx.formatValueToDisplay(), mParsedTx.getSymbol());
+
+            if (mTxInfo.txType == TransactionType.ERC721_TRANSFER_FROM
+                    || mTxInfo.txType == TransactionType.ERC721_SAFE_TRANSFER_FROM) {
+                amountText = Utils.tokenToString(mParsedTx.getErc721BlockchainToken());
+                amountFiat.setVisibility(View.GONE); // Display NFT values in the future
+            }
+            amountAsset.setText(amountText);
+        }
+
+        String accountName = mWalletListItemModel.getAccountInfo().name;
+        if (mParsedTx.getSender() != null
+                && !mParsedTx.getSender().equals(mParsedTx.getRecipient())) {
+            String recipient =
+                    TextUtils.isEmpty(mParsedTx.getRecipient()) ? "..." : mParsedTx.getRecipient();
```

Comment:
Instead of using three separate dots `...` there's a specific character called ellipsis that usually is visually better and is specific for this purpose ->  `…`.

Reply:
I think it's okay and better in terms of visibility though we can changes this across codebase in a separate issue if required.